lib/Test/Nginx/Util.pm

```diff
@@ -2364,6 +2367,28 @@ retry:
             #warn "pid file not found";
         }
     }
+
+    if (defined $block->shutdown_error_log) {
+        if ($UseHup) {
+            warn "skip the shutdown_error_log tests under the HUP reload mode\n";
+            return;
+        }
+
+        if ($ENV{TEST_NGINX_NO_CLEAN}) {
+            warn "skip the shutdown_error_log tests under set TEST_NGINX_NO_CLEAN flag\n";
```

Comment:
By "skipping tests", I mean using the `SKIP: { ... }` block so that we do not break the test plan in those test modes.